packages/graph-explorer/src/connector/queries.ts

```diff
@@ -0,0 +1,74 @@
+import { queryOptions } from "@tanstack/react-query";
+import {
+  Explorer,
+  NeighborsRequest,
+  NeighborsResponse,
+  VertexId,
+  typeofVertexId,
+} from "./useGEFetchTypes";
+
+function exponentialBackoff(attempt: number): number {
```

Comment:
we already have `retryFetch` in proxy. How it will work together?

Reply:
The retry in the proxy is disabled (max of 1). If it was to be enabled, each retry of the client would result in retries in the proxy.

Reply:
The retries in the client affect everyone, even those who do not use the proxy. So I would prefer to do the retries there. If we require a server in the future, then the retries should move to the server for performance reasons.